de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/ProjectNegotiation.java

```diff
@@ -71,6 +73,8 @@
 
     protected TransferType transferType;
 
+    protected IReferencePointManager referencePointManager;
+
     public ProjectNegotiation(final String id, final JID peer,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 10 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=258&issues=AWbe6KEnSgN4uCclR4y6&open=AWbe6KEnSgN4uCclR4y6)

[ISSUE_KEY]: AWbe6KEnSgN4uCclR4y6